Sources/BraintreeCore/BTGraphQLHTTP.swift

```diff
@@ -203,7 +203,14 @@ class BTGraphQLHTTP: BTHTTP {
         let errorTree = BTGraphQLErrorTree(message: "Input is invalid")
 
         for errorJSON in body["errors"].asArray() ?? [] {
-            guard let inputPath = errorJSON["extensions"]["inputPath"].asStringArray() else { continue }
+            guard let inputPath = errorJSON["extensions"]["inputPath"].asStringArray() else {
```

Comment:
We want to add specific error messaging for the validation errors coming from the payment context service, and those do not have an `inputPath`